docs/source/support/overview.md

```diff
@@ -0,0 +1,14 @@
+# Support overview
+
+## Get support
+
+ScyllaDB provides administrators with [paid support](https://www.scylladb.com/product/support/#enterprise-support), including Scylla Operator.
+
+## Troubleshooting issues
+
+* [Installation issues](troubleshooting/installation.md)
```

Comment:
nit: I think pointing here to the index page of troubleshooting instead having a bullet list of subpages would be more extensible in future.

Reply:
fixed